### ISSUE TITLE
Parameter's model is not shown when the parameter is defined in parameters section

### ISSUE THREAD

**Issue by gaaf** [swagger-api/swagger-ui#1399]
When a parameter is defined as a reference to an item in the 'parameters' section (`{ "$ref": "#/parameters/Param" }`), the model isn't shown in the ui. The url to the definition is shown instead.

The example below shows this problem under the '/ref' path. For reference, the same parameter is defined directly under the path '/direct', for which the model is shown in the ui.

This is on 2.1.0+, @d75505fb73a6ac4ecbeabc47bd5a42b0f4d3f270

``` json
{
    "swagger": "2.0",
    "basePath": "/", 
    "schemes": [
        "https"
    ],
    "info": {
        "version": "2.1.0",
        "title": "Parameter bug demo"
    },
    "parameters": {
        "Param": {
            "name": "param",
            "in": "body",
            "required": true,
            "schema": {
                "$ref": "#/definitions/Param"
            }
        }
    },
    "paths": {
        "/direct": {
            "post": {
                "parameters": [
                    {
                        "name": "param",
                        "in": "body",
                        "required": true,
                        "schema": {
                            "$ref": "#/definitions/Param"
                        }
                    }
                ],
                "responses": {
                    "default": {
                        "description": "Return"
                    }
                }
            }
        },
        "/ref": {
            "post": {
                "parameters": [
                    { "$ref": "#/parameters/Param" }
                ],
                "responses": {
                    "default": {
                        "description": "Return"
                    }
                }
            }
        }
    },
    "definitions": {
        "Param": {
            "type": "object",
            "properties": {
                "test": {
                    "type": "string"
                }
            }
        }
    }
}
```

**Comment by webron** [comment]
Looks fine in master:

![screenshot 2015-06-22 11 47 23](https://cloud.githubusercontent.com/assets/241629/8278496/85ac0522-18d4-11e5-88ad-74ac0a75566a.png)
![screenshot 2015-06-22 11 47 36](https://cloud.githubusercontent.com/assets/241629/8278498/89c8d0cc-18d4-11e5-8351-939f7c7b0144.png)

**Comment by gaaf** [comment]
For me it looks like this on master (in Chromium):

![swagger-ui- 1399-1](https://cloud.githubusercontent.com/assets/5906877/8278760/70bea936-18ce-11e5-82e8-04b10ddfd36e.png)

**Comment by webron** [comment]
Can you try clearing your cache? Tested it with Chromium, still looks fine.

**Comment by webron** [comment]
Never mind, managed to reproduce it. The behavior is different if it is embedded or hosted (only occurs when hosted).

**Comment by gaaf** [comment]
The fix for #1423 inspired me. I changed all occurrences of the same code pattern in swagger-ui and swagger-js accordingly. That fixed this issue for me.

**Comment by webron** [comment]
This should be resolved in 3.0.
